lib/tiled/object_layer.rb

```diff
@@ -16,7 +16,7 @@ def initialize(map)
     # @return [Tiled::ObjectLayer] self
     def from_xml_hash(hash)
       raw_attributes = hash[:attributes]
-      raw_attributes['color'] = Color.from_tiled_rgba(raw_attributes['color'])
+      raw_attributes['color'] &&= Color.from_tiled_rgba(raw_attributes['color'])
```

Comment:
`Color.from_tiled_rgba` will always return instance of `Tiled::Color`, so there is no need of `&&=`

Reply:
Ah, I had a nil error when I was first testing your changes but it must have been something else. Removing.